pkg/commands/image/patch.go

```diff
@@ -103,7 +103,8 @@ kp image patch my-image --tag my-registry.com/my-repo --blob https://my-blob-hos
 			return nil
 		},
 	}
-	cmd.Flags().StringArrayVar(&factory.AdditionalTags, "additional-tag", []string{}, "additional tags to push the OCI image to")
+	cmd.Flags().StringArrayVar(&factory.AdditionalTags, "additional-tag", []string{}, "adds additional tags to push the OCI image to")
+	cmd.Flags().StringArrayVar(&factory.ReplaceAdditionalTags, "replace-additional-tag", []string{}, "replaces all additional tags to push the OCI image to")
```

Comment:
wdyt of adding a blurb to the long description about the use case for this flag? To me, `replaces all additional tags to push the OCI image to` doesn't instinctively tell me why I should use this flag.